playwright/components/organisms/emailSignUpForm.spec.js

```diff
@@ -89,7 +88,7 @@ test.describe('Email Signup Form component', () => {
     await page.locator('[data-testid="EmailSignUpForm-example-1"] #email').fill(''); // clear the email field
     await page.locator('[data-testid="EmailSignUpForm-example-1"] #email').type('Test0-9!#$%&\'*+/=?^_{|}~-@comicrelief_9-8.com.uk');
     await page.locator('[data-testid="EmailSignUpForm-example-1"] #last-name').click();
-    await expect(page.locator('label[for="email"]  div ~ span')).toBeHidden('');
+    await expect(page.locator('label[for="email"]  div ~ span')).toContainText('Please enter a valid email address');
```

Comment:
Hi @KrupaPammi - was kinda confused about this; given the test it's doing here, using 'toBeHidden' here feels wrong, as it **_should_** be showing an error, no?

I'm also very confused as to why me updating Yup has suddenly tripped this...